### ISSUE TITLE
NEI crash with 1.10.2

### ISSUE THREAD

**Issue by blockbreaker1361** [TheCBProject/NotEnoughItems#9]
I know that the 1.9.4 version is not an official build for 1.10.2, but I wanted to post this crash report anyway. NEI will work fine if you are in creative mode, but if you attempt to open it in survival, it instantly crashes. I am using the latest of everything including JEI. Here's the crash report
http://pastebin.com/q7QYBdw6

**Comment by covers1624** [comment]
Yeah this is a known issue, at the start of August i will release for 1.10 officially as that is when BTM finishes.

**Comment by blockbreaker1361** [comment]
OK. Thanks.
